### PR TITLE
Ginkgo: Improved failed report on Kubernetes

### PR DESCRIPTION
**Pull request by eloycoto**
- Create Kubernetes folders with version
- Added -o wide in Kubernetes commands

Signed-off-by: Eloy Coto <eloy.coto@gmail.com>

### CODE REVIEW

File: test/helpers/utils.go [hunk 1]
```diff
@@ -152,8 +152,14 @@ func Fail(description string, callerSkip ...int) {
 // If the directory cannot be created it'll return an error
 func ReportDirectory() (string, error) {
 	testDesc := ginkgo.CurrentGinkgoTestDescription()
-	testPath := fmt.Sprintf("%s%s/",
+	prefix := ""
+	if strings.HasPrefix(strings.ToLower(testDesc.FullTestText), K8s) {
+		prefix = fmt.Sprintf("%s-", strings.Replace(GetCurrentK8SEnv(), ".", "", -1))
+	}
+
+	testPath := fmt.Sprintf("%s%s%s/",
```
Comment (aanm) [inline test/helpers/utils.go]: Try use filepath.Join

**Review by aanm — CHANGES_REQUESTED** [review]

**Review by ianvernon — APPROVED** [review]
LGTM, but as @aanm said, please use `filepath.Join`.

**Comment by eloycoto** [comment]
changes are done, and PR rebased. 

Regards

**Review by ianvernon — APPROVED** [review]

**Comment by ianvernon** [comment]
test-me-please

**Comment by eloycoto** [comment]
The problem in the failing test: 

```
19:13:05 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: ----- 10-proxy.sh: main/proxy_test: beginning proxy test
19:13:05 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: ----- 10-proxy.sh: main/proxy_test/monitor_clear: clearing monitor
19:13:05 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: ----- 10-proxy.sh: main/proxy_test: trying to reach server IPv4 at http://10.11.193.100:80/public from client (expected: 200)
19:13:05 [Runtime Tests with Envoy] ==> cilium-master-build-Cilium-Bash-Tests-9-envoy: e4e3e63a2a591d595585ebe14678542cdc779f6af8d3747df47148e4ee7a773f
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: Internal Server Error
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: ------------------------------------------------------------------------
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9:                             Test Failed
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: GET /public, unexpected return 500 != 200
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: 
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: ------------------------------------------------------------------------
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: #kernel
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: IpInReceives                    22                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: IpInDelivers                    22                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: IpOutRequests                   23                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpActiveOpens                  2                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpPassiveOpens                 2                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpEstabResets                  1                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpInSegs                       22                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpOutSegs                      23                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpOutRsts                      1                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPPrequeued              1                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPDirectCopyFromPrequeue 1216               0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPHPHits                 1                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPHPHitsToUser           1                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPPureAcks               7                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPHPAcks                 2                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPAbortOnData            1                  0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: TcpExtTCPOrigDataSent           11                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: IpExtInOctets                   3967               0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: IpExtOutOctets                  4600               0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: IpExtInNoECTPkts                22                 0.0
19:13:06 [Runtime Tests] ==> cilium-master-build-Cilium-Bash-Tests-9: 
```

Looks like that it's a random fail no related with the new Jenkins process.

**Comment by eloycoto** [comment]
test-me-please

**Review by aanm — APPROVED** [review]

**Pull request merged**